GNUmakefile

```diff
@@ -158,6 +158,7 @@ clang:
 ## Coverage/profiling
 
 .PHONY: coverage
+coverage: export PCMK__MAKE_COVERAGE = 1
```

Comment:
I've got mixed feelings about this commit. It makes the tests more difficult to follow for a very narrow benefit.

I think I'd rather add unit tests for the not-really-covered functions, even if that takes a while. It would be helpful to start by figuring out what those functions are, so we can estimate how much work it would be.

Reply:
It's a lot of code - basically, anything in the setup and teardown functions in lib/pengine/tests/native/*c.  There's pe_new_working_set, set_working_set_defaults, pe__set_working_set_flags, pe_free_working_set, pe_find_node, and everything those call.  None of those are especially bad.

The big one is cluster_status.

We could definitely sneak up on covering those functions, and perhaps that should be the next unit test related thing I do.  But cluster_status does a ton and it basically makes it impossible to see what XML- and cluster-related code we have left to cover.  We could consider this patch as a temporary measure while we work on it, at least.  As it stands right now, the coverage numbers just don't mean anything to me.  I can't tell at a glance if a covered line is something we've taken care of already, or if it's just incidentally covered because of these setup functions.

Reply:
I'm leaning to it not being worth it at this point since our coverage is so low that knowing it isn't particularly helpful. We could maybe schedule a "unit test week" once or twice a year where we all knock out as many of the lowest-level functions as possible. Reid will get a few as part of his onboarding.

Reply:
Updated and dropped this patch.  I think adding testing in this area in particular is going to require a lot of work, because it's a lot of pretty big functions.  We'll be better off for having it in the future though.